src/video.c

```diff
@@ -1275,9 +1275,8 @@ int video_update(struct video *v, const char *peer)
 
 	if (!sc) {
 		info("video: video stream is disabled..\n");
-		video_stop_source(v);
-		video_stop_display(v);
-		return err;
+		video_stop(v);
+		return 0;
```

Comment:
perhaps this can be split into two PRs, one for audio and one for video ?

Reply:
- https://github.com/baresip/baresip/pull/2982